igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildCache.groovy

```diff
@@ -38,16 +38,18 @@ class JenkinsCache {
     @Value('${spinnaker.jedis.prefix:igor}')
     String prefix
 
+    String id = 'builds'
+
     List<String> getJobNames(String master) {
         Jedis resource = jedisPool.resource
-        List<String> jobs = resource.keys("${prefix}:${master}:*").collect { extractJobName(it) }.sort()
+        List<String> jobs = resource.keys("${prefix}:${id}:${master}:*").collect { extractJobName(it) }.sort()
```

Comment:
Jenkins cache will be nuked, so I think all jenkins builds in spinnaker will trigger a new build event to echo. So I guess this means it will have catastrophic consequences for existing setups.